### ISSUE TITLE
Common issue: not logged in with the motus.sample account

### ISSUE THREAD

**Issue by leberrigan** [MotusDev/Motus-TO-DO#624]
It appears that the latest version of the Motus R package (v 3.0.0) does not provide access to the sample data set. 

Attempts by both Josh and I to download from project 176 resulted in zero detections and was only permitted after logging in as user with permissions with project 176 (user 'testaccount' was unsuccessful).

**Comment by steffilazerte** [comment]
I can't replicate this problem, are we using the same sample data account?

As far as I know the sample data set is accessed via the "motus.sample" account (with pass also "motus.sample").

**Comment by jsayers-bsc** [comment]
Yes, that's the issue apparently!

I guess the suggestion then would be that maybe this error could be mentioned in the "troubleshooting" section, or as a prominent sidebar when discussing accessing the sample data. E.g. "Ensure you are logged in with the 'motus.sample' account and not your own account" or "If you receive 0 records then you are likely using your own login information rather than the 'motus.sample'" login. I imagine that's a pretty common mistake!

**Comment by steffilazerte** [comment]
Okay! I've made the following changes:
- `motus` package (beta3) examples all include a reminder of the username/password
- Motus R book has a reminder at the first `tagme()` in each chapter (it was there already for most of them, but I added a couple that were missing)
- Motus R book (beta) has a reference to this in the [troubleshooting section](https://motus.org/MotusRBook/appendixB.html#common-error-messages-and-solutions) (as recommended by @jsayers-bsc) [*Currently this isn't in the online version, I'm not sure how/when the online version gets updated? The change to the text can be seen [here](https://github.com/MotusWTS/MotusRBook/blob/beta/10-AppendixB.Rmd#L44-L45)*]

If @jsayers-bsc and @leberrigan are happy with these changes, I'll close this issue.
